libs/h3/src/main/java/org/elasticsearch/h3/Vec2d.java

```diff
@@ -146,11 +146,12 @@ public LatLng hex2dToGeo(int face, int res, boolean substrate) {
 
         // adjust theta for Class III
         // if a substrate grid, then it's already been adjusted for Class III
-        if (substrate == false && H3Index.isResolutionClassIII(res)) theta = posAngleRads(theta + Constants.M_AP7_ROT_RADS);
+        if (substrate == false && H3Index.isResolutionClassIII(res)) {
+            theta = posAngleRads(theta + Constants.M_AP7_ROT_RADS);
+        }
 
         // find theta as an azimuth
         theta = posAngleRads(faceAxesAzRadsCII[face][0] - theta);
```

Comment:
Not sure why line was removed, liked it a little better before.

Reply:
The coding standards are that all if statements should have brackets.

Reply:
My comment was about the line below this, which was a blank line removed. Not about the lines above with the brackets.